src/views/CharacterListView.js

```diff
@@ -2,20 +2,18 @@ import React from "react";
 import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
-// import actions
+import { getCharacters } from "../actions"
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
 
   componentDidMount() {
-    // call our action
+    this.props.getCharacters()
   }
 
   render() {
     if (this.props.fetching) {
       // return something here to indicate that you are fetching data
+      return <div> ...loading </div>
     }
```

Comment:
```suggestion
    }
    else if (this.props.error) {
      return <div>{this.props.error.message}</div>
    }
```
I would just add some error handling here